rhel7/profiles/cis.profile

```diff
@@ -581,6 +581,7 @@ selections:
     - sshd_disable_x11_forwarding
     
     ### 5.2.5 Ensure SSH MaxAuthTries is set to 4 or less (Scored)
+    - sshd_max_auth_tries_value=4
```

Comment:
Can the OVAL be updated to use this variable?

Reply:
The OVAL already uses this as a variable: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/ssh/ssh_server/sshd_set_max_auth_tries/oval/shared.xml#L44

I just wanted to make explicit similar to the CIS RHEL8 profile: https://github.com/ComplianceAsCode/content/blob/master/rhel8/profiles/cis.profile#L763

Reply:
The OVAL is using it already:
https://github.com/ComplianceAsCode/content/blob/dcefd47e94095cbb39059f5d0ec9ef42593ae595/linux_os/guide/services/ssh/ssh_server/sshd_set_max_auth_tries/oval/shared.xml#L44